diff -Naur camE-1.9/webcam.c camE-1.9-fix-curl-issue/webcam.c
--- camE-1.9/webcam.c	2004-06-26 01:51:29.000000000 +0200
+++ camE-1.9-fix-curl-issue/webcam.c	2007-01-12 15:36:28.000000000 +0100
@@ -988,7 +988,9 @@
     if (ftp_debug)
       curl_easy_setopt(curl_handle, CURLOPT_VERBOSE, 1);
     else
+#if LIBCURL_VERSION_NUM < 0x71000
       curl_easy_setopt(curl_handle, CURLOPT_MUTE, 1);
+#endif
 
     curl_easy_setopt(curl_handle, CURLOPT_UPLOAD, 1);
 
